cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -148,6 +273,15 @@ OPTIONS "{" btree-option: btree-value[, ...] "}"
 a|
 Node key and uniqueness constraints with b-tree options are deprecated and need to be migrated before upgrading to 5.0.
 
+Replaced by:
+[source, cypher, role="noheader"]
+----
+CREATE CONSTRAINT
+...
+OPTIONS "{" range-option: range-value[, ...] "}"
+----
+Constraints with the `lucene+native-3.0` index provider or spatial index configuration settings have no replacement.
```

Comment:
Well that is not quite true, it can be considered as a range constraint and then a text or point index on the same schema

Reply:
will see if can reformulate it a bit